res/css/views/rooms/_EventTile.scss

```diff
@@ -1111,39 +1121,34 @@ $threadInfoLineHeight: calc(2 * $font-12px); // See: _commons.scss
             &.mx_EventTile_emote {
                 padding-top: $spacing-8; // add a bit more space for emotes so that avatars don't collide
 
-                &.mx_EventTile_continuation {
-                    padding-top: 0;
-
-                    .mx_EventTile_line,
-                    .mx_EventTile_reply {
-                        padding-top: 0;
-                        padding-bottom: 0;
-                    }
-                }
-
                 .mx_EventTile_avatar {
-                    top: 2px;
+                    top: var(--MatrixChat_useCompactLayout-top-avatar);
                 }
 
                 .mx_EventTile_line,
                 .mx_EventTile_reply {
-                    padding-top: 0;
                     padding-bottom: 1px;
                 }
+
+                &.mx_EventTile_continuation {
+                    .mx_EventTile_line,
+                    .mx_EventTile_reply {
+                        padding-bottom: var(--MatrixChat_useCompactLayout_line-spacing-block);
+                    }
+                }
             }
 
-            .mx_EventTile_avatar {
-                top: 2px;
+            // Cascading - apply zero padding to every element including mx_EventTile_emote
+            &.mx_EventTile_continuation {
+                padding-top: var(--MatrixChat_useCompactLayout_line-spacing-block);
```

Comment:
This declaration ensures zero padding-top after moving `!important` flag on `_EventTile.scss`.